services/wallet/router/pathprocessor/processor_swap_paraswap.go

```diff
@@ -26,6 +26,7 @@ type SwapParaswapTxArgs struct {
 type SwapParaswapProcessor struct {
 	paraswapClient *paraswap.ClientV5
 	priceRoute     paraswap.Route
+	slippageBP     uint // in basis points
```

Comment:
Suggestion: instead of the comment we could rename to `slippageBasisPoints`.

Reply:
That's why we have the suffix `BP` 😄 is that enough for you or would you still like to have a full name?

Reply:
@saledjenic, no strong opinion, but usually if the code requires a comment to be clear, then maybe the code could be renamed and the comment removed so that the code is a bit more self-documenting. For example, if the comment is needed here, perhaps it also means a dev wouldn't immediately understand what it is in other parts of the codebase that don't have the comment. But the suffix is probably fine for those touching swap code and understand what it means 👍🏼